src/arns.lua

```diff
@@ -418,10 +418,11 @@ end
 
 ---Calculates the lease fee for a given base fee, years, and demand factor
 --- @param baseFee number The base fee for the name
---- @param years number The number of years
+--- @param years number|nil The number of years
 --- @param demandFactor number The demand factor
 --- @return number leaseFee - the lease fee
 function arns.calculateLeaseFee(baseFee, years, demandFactor)
+	assert(years, "Years is required for lease")
```

Comment:
looks like this was previously removed from `getTokenCost` switch statement? we were asserting it existed there before calling this

Reply:
seems so. might be more assertions around this that aren't being met -- this is just one I ran into here